docs/annotations.md

```diff
@@ -183,3 +184,37 @@ spec:
           serviceName: go-server-service
           servicePort: 80
 ```
+
+## Use Private IP
+
+This annotation allows to specify whether to expose this endpoint on Private IP of Application Gateway.
+
+> **Note**
+1) Application Gateway doesn't support multiple IPs on the same port (example: 80/443). So, if you have one ingress with annotation `use-private-ip: "false"` and other with `use-private-ip: "true"` on `HTTP`, they will fail the update on the Gateway.
+2) In case where Application Gateway doesn't have a private IP, specific ingresses with `use-private-ip: "true"` be pruned out. This will reflected in the controller logs and ingress events for those ingresses with `NoPrivateIP` .
```

Comment:
I wonder if we should rephrase this as `App Gateways that don't have a private IP and are annotated with … will be ignored`.